time_series_export/slack_ts_export.cpp

```diff
@@ -0,0 +1,43 @@
+#include <string>
+
+#include "glog/logging.h"
+
+#include "serenity/agent_utils.hpp"
+
+#include "slack_ts_export.hpp"
+
+namespace mesos {
+namespace serenity {
+
+Try<Nothing> SlackTimeSeriesExporter::consume(const Resources& resources) {
+  double_t cpus = 0.0;
+
+  Option<double_t> cpus_option = resources.cpus();
+  if (cpus_option.isSome()) {
+    cpus = cpus_option.get();
+  }
+
+  Try<std::string> hostname = AgentInfo::GetHostName();
+  if (hostname.isError()) {
+    LOG(ERROR) << "ResourceUsageTimeSeriesExporter: cannot get hostname";
+    return Nothing();  // Do not ruin serenity because small thing
```

Comment:
s/Do not ruin serenity because small thing/Do not cause failure in pipeline due to stats reporting failure/